deeds-tenant-service/pom.xml

```
@@ -81,5 +81,29 @@
       <artifactId>mockito-core</artifactId>
       <scope>test</scope>
     </dependency>
+      <dependency>
+          <groupId>org.exoplatform.core</groupId>
+          <artifactId>exo.core.component.organization.api</artifactId>
+      </dependency>
```

```suggestion
      <dependency>
        <groupId>org.exoplatform.core</groupId>
        <artifactId>exo.core.component.organization.api</artifactId>
        <scope>provided</scope>
      </dependency>
```
extra spaces + missing scope provided.